From 67137663f67afa6c7ba9ef29f9b9c2e9288bb0bd Mon Sep 17 00:00:00 2001
From: prumian <100143890+prumian@users.noreply.github.com>
Date: Tue, 22 Feb 2022 07:47:39 +0100
Subject: [PATCH] hwdb: add CST Laser Trackball (#22583)

Set the default to 800DPI as this is what user manual suggests.

(cherry picked from commit d9ab110ddc453cac40a0b36cee5e5dc272e7aec7)

Related: #2087778
---
 hwdb.d/70-mouse.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 66e74f667c..723d145e34 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -166,6 +166,14 @@ mouse:bluetooth:v05acp030d:name:*:*
 mouse:usb:v04f2p0963:name:Chicony 2.4G Multimedia Wireless Kit:*
  MOUSE_DPI=1000@142
 
+##########################################
+# CST
+##########################################
+
+# CST Laser Trackball
+mouse:usb:v5332p1400:name:Clearly Superior Technologies. CST Laser Trackball:*
+ MOUSE_DPI=400@1000 *800@1000 1600@1000
+
 ##########################################
 # Dell
 ##########################################
